src/config_maker/view/dialogs/quantization_config_dialog.py

```diff
@@ -0,0 +1,615 @@
+# pylint: disable-next=E0401
+from tags import HEADER_POT_PARAMS_TAGS, HEADER_MODEL_PARAMS_ENGINE_TAGS, \
+    HEADER_MODEL_PARAMS_MODEL_TAGS, HEADER_MODEL_PARAMS_COMPRESSION_COMMON_TAGS, \
+    HEADER_DQ_PARAMS_TAGS, HEADER_AAQ_PARAMS_TAGS, HEADER_INDEPENDENT_PARAMS_TAGS, \
+    CONFIG_TARGET_DEVICE_TAG, CONFIG_ALGORITHM_NAME_TAG, CONFIG_PRESET_TAG, \
+    CONFIG_STAT_SUBSET_SIZE_TAG, CONFIG_WEIGHTS_BITS_TAG, CONFIG_WEIGHTS_MODE_TAG, \
+    CONFIG_WEIGHTS_GRANULARITY_TAG, CONFIG_WEIGHTS_LEVEL_LOW_TAG, CONFIG_WEIGHTS_LEVEL_HIGH_TAG, \
+    CONFIG_WEIGHTS_MAX_TYPE_TAG, CONFIG_WEIGHTS_MAX_OUTLIER_PROB_TAG, CONFIG_ACTIVATIONS_BITS_TAG, \
+    CONFIG_ACTIVATIONS_MODE_TAG, CONFIG_ACTIVATIONS_GRANULARITY_TAG, CONFIG_ACTIVATIONS_PRESET_TAG, \
+    CONFIG_ACTIVATIONS_MIN_CLIPPING_VALUE_TAG, CONFIG_ACTIVATIONS_MIN_AGGREGATOR_TAG, \
+    CONFIG_ACTIVATIONS_MIN_TYPE_TAG, CONFIG_ACTIVATIONS_MIN_OUTLIER_PROB_TAG, \
+    CONFIG_ACTIVATIONS_MAX_CLIPPING_VALUE_TAG, CONFIG_ACTIVATIONS_MAX_AGGREGATOR_TAG, \
+    CONFIG_ACTIVATIONS_MAX_TYPE_TAG, CONFIG_ACTIVATIONS_MAX_OUTLIER_PROB_TAG
+
+
+class QuantizationConfigDialog(QDialog):
+    def __init__(self, parent, models, data):
+        super().__init__(parent)
+        self.__title = 'Information about model'
+        self.__q_method_dependent_params = {
+            'DefaultQuantization': DefaultQuantizationDialog(self),
+            'AccuracyAwareQuantization': AccuracyAwareQuantizationDialog(self)
+        }
+        self.__q_method_independent_params = IndependentParameters(
+            self, models, data, self.__q_method_dependent_params.keys(), self.__q_method_choice)
+        self.__selected_q_method = 'DefaultQuantization'
+        self.__pot_params_tags = HEADER_POT_PARAMS_TAGS
+        self.__model_params_tags = HEADER_MODEL_PARAMS_MODEL_TAGS + \
+            HEADER_MODEL_PARAMS_ENGINE_TAGS + HEADER_MODEL_PARAMS_COMPRESSION_COMMON_TAGS + []
+        self.tags = [*self.__pot_params_tags, *self.__model_params_tags]
+        self.__init_ui()
+        self.__q_method_independent_params.switch_engine_type(self.__selected_q_method)
+
+    def __init_ui(self):
+        self.setWindowTitle(self.__title)
+        self.__create_layout()
+
+    def __create_layout(self):
+        layout = QGridLayout()
+        idx = 0
+        independent_idx = self.__q_method_independent_params.attach_to_layout(layout)
+        dict_q_method_idx = dict.fromkeys(self.__q_method_dependent_params.keys(), idx)
+        for key in self.__q_method_dependent_params:
+            dict_q_method_idx[key] = self.__q_method_dependent_params[key].attach_to_layout(layout, False)
+        self.__q_method_dependent_params['DefaultQuantization'].show()
+        dependent_idx = max(*dict_q_method_idx.values())
+        row_idx = max(independent_idx, dependent_idx)
+        ok_btn = QPushButton('Ok')
+        cancel_btn = QPushButton('Cancel')
+        ok_btn.clicked().connect(self.accept)
+        cancel_btn.clicked().connect(self.reject)
```

Comment:
```suggestion
        ok_btn.clicked.connect(self.accept)  # pylint: disable=E1120
        cancel_btn.clicked.connect(self.reject)  # pylint: disable=E1120
```

Поправь так, пожалуйста, везде в PR.

Issue https://github.com/PyCQA/pylint/issues/6464